src/wallets/history/Transaction.ts

```diff
@@ -0,0 +1,86 @@
+import {Ord} from 'fp-ts/Ord'
+import {pipe} from 'fp-ts/pipeable'
+import {array, option, ord} from 'fp-ts'
+import {Wei} from '../../common/units'
+import {prop} from '../../shared/utils'
+
+export type TransactionStatus =
+  | 'pending'
+  | 'confirmed'
+  | 'persisted_depth'
+  | 'persisted_checkpoint'
+  | 'failed'
+const TransactionStatus = (() => {
```

Comment:
Is it `any`? AFAIK we have enabled `noImplicitAny` to prevent such issues. I can change casing so it will follow conventions like:
```
type Foo = 'foo'
const foo = {//operations on Foo}
```

I would like to keep grouping methods in object since it makes it really nice to use in longer term as well as makes it nice and easy to use `fp-ts`'s type classes approach as one/two imports are needed for everything directly connected with given type

Reply:
Okay, it seems that just my editor is confused, not sure why. This is what I see:
![image](https://user-images.githubusercontent.com/4038239/102906806-72a1c980-4475-11eb-98d7-d007e139b204.png)